### PR TITLE
Change single job pipeline to use workflow events

### PR DESCRIPTION
**Pull request by karenc**
The single job pipeline was using slack as the trigger.  Change it to
use workflow events instead.

### CODE REVIEW

File: bakery/pipeline.yml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 ---
 resource_types:
-  - name: slack
+  - name: workflow-events
     type: docker-image
     source:
-      repository: karenc/concourse-slack-resource
+      repository: openstax/workflow-event-resource
```
Comment (c0ldcalamari) [inline bakery/pipeline.yml:6]: We have the updated resource now, 
https://hub.docker.com/r/openstax/output-producer-resource. Should we use it?

**Review by c0ldcalamari — APPROVED** [review]

File: bakery/tasks/fetch-book.yml [hunk 2]
```diff
@@ -18,5 +18,11 @@ run:
...
-      mkdir -p "$book_dir"
-      neb get -r -d "$book_dir/raw" "$(cat ../book/server)" "$(cat ../book/collection_id)" "$(cat ../book/version)"
+      mkdir -p "$book_dir" ~/.config/
+      server="$(cat ../book/server)"
+      cat >~/.config/nebuchadnezzar.ini <<EOF
+      [settings]
+      [environ-$server]
+      url = https://$server
+      EOF
+      yes | neb get -r -d "$book_dir/raw" "$(cat ../book/server)" "$(cat ../book/collection_id)" "$(cat ../book/version)"
```
Comment (tomjw64) [inline bakery/tasks/fetch-book.yml:28]: I didn't know about `yes` until this. I tried it and laughed as a million 'y's showed up in my terminal :joy:

Comment (karenc) [reply bakery/tasks/fetch-book.yml:28]: hahahaha it's really useful!
I guess we could've add another option to neb so we don't need to use `yes` :p

File: bakery/pipeline.yml (hunk 1 above)
Comment (karenc) [reply bakery/pipeline.yml:6]: I'm going to update both pipelines to use output-producer-resource after this.

**Pull request merged**